src/index.ts

```diff
@@ -54,6 +56,34 @@ function getRecordExtractorSource(): string {
 }`;
 }
 
+function addComment(crawlerId: string): void {
+  try {
+    const pathArray = CRAWLER_API_BASE_URL.split('/');
+    const protocol = pathArray[0];
+    const host = pathArray[2];
+    const baseUrl = `${protocol}//${host}`;
+
+    const message = `<p>Check Crawler: ${baseUrl}/admin/crawlers/${crawlerId}/overview</p>
```

Comment:
Not sure how it renders but I'm sure we can do better.
At least put some proper links ☺️